aspnetcore/tutorials/index.md

```diff
@@ -20,22 +20,31 @@ The following step-by-step guides for developing ASP.NET Core applications are a
 [Razor Pages](xref:mvc/razor-pages/index) is the recommended approach to create a Web UI with ASP.NET Core 2.0.
 
 * [Introduction to Razor Pages in ASP.NET Core](xref:mvc/razor-pages/index)
-* [Create a Razor Pages web app with ASP.NET Core on Windows](xref:tutorials/razor-pages/index)
-* [Create a Razor Pages web app with ASP.NET Core on Mac](xref:tutorials/razor-pages-mac/index)  
-* [Create a Razor Pages web app with ASP.NET Core with VS Code](xref:tutorials/razor-pages-vsc/index) 
-* [Create an ASP.NET Core MVC web app with Visual Studio on Windows](first-mvc-app/index.md)
-* [Create an ASP.NET Core MVC web app with Visual Studio on Mac](first-mvc-app-mac/index.md)
-* [Create an ASP.NET Core MVC web app with Visual Studio Code on Mac or Linux ](first-mvc-app-xplat/index.md)
+* Create a Razor Pages web app with ASP.NET Core
```

Comment:
I suggest changing this title to "Create an ASP.NET Core Razor Pages web app". This makes it more symmetrical with the "Create an ASP.NET Core MVC web app" title below it.